<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="YOGITA" failures="0" tests="1" name="testNG1_Test.CF_TC512_ValidateErrorMessage" time="12.566" errors="0" timestamp="2024-01-17T13:04:43 IST" skipped="0">
  <testcase classname="testNG1_Test.CF_TC512_ValidateErrorMessage" name="CF_TC512_ValidateErrorMessageForPinCodeField" time="12.566">
    <system-out>
      <![CDATA[TC CF_TC512_ValidateErrorMessageForPinCodeField excecution is started
error message is display 
TC CF_TC512_ValidateErrorMessageForPinCodeField is completed successfully]]>
    </system-out>
  </testcase> <!-- CF_TC512_ValidateErrorMessageForPinCodeField -->
  <system-out>
    <![CDATA[Browser luanching
Click on gender button
Click on Next button
handeling dropDown 
Click on next button
entering mobile number
Click on continue button
TC CF_TC512_ValidateErrorMessageForPinCodeField excecution is started
error message is display 
TC CF_TC512_ValidateErrorMessageForPinCodeField is completed successfully
Browser closing ]]>
  </system-out>
</testsuite> <!-- testNG1_Test.CF_TC512_ValidateErrorMessage -->
